spec/moonshot/plugins/backup_spec.rb

```diff
@@ -0,0 +1,98 @@
+describe Moonshot::Plugins::Backup do
+  let(:hooks) do
+    [
+      :pre_create,
+      :post_create,
+      :pre_update,
+      :post_update,
+      :pre_delete,
+      :post_delete,
+      :pre_status,
+      :post_status,
+      :pre_doctor,
+      :post_doctor
+    ]
+  end
+
+  describe '#new' do
+    subject { Moonshot::Plugins::Backup }
+
+    it 'should yield self' do
+      expect do |b|
```

Comment:
This also makes sense, what I was trying to achieve here was to inspect whether it yields at all or not.